Source/Core/Core/ConfigManager.cpp

```diff
@@ -743,14 +746,15 @@ void SConfig::LoadSettingsFromSysconf()
 
 void SConfig::ResetRunningGameMetadata()
 {
-  SetRunningGameMetadata("00000000", 0, 0);
+  SetRunningGameMetadata("00000000", 0, 0, Core::TitleDatabase::TitleType::Other);
```

Comment:
The commit that changes this comes before the commit that added the enum. (Or is GitHub showing commits in an unintuitive order again?)